server/services/nextPayment.service.js

```diff
@@ -241,6 +242,14 @@ export const sendReminder = async () => {
       await createNotification(NOTIFICATIONS.PAYMENT_REMINDER, reminder.userId, {
         actionId: reminder.offerId,
       });
+
+      const message = `This is a quick reminder that the periodic payment on your property ${
+        reminder.propertyInfo.name
+      }, is due on ${convertDateToShortHumanFormat(reminder.expiresOn)}.`;
```

Comment:
Not sure why you made this error again. I thought we agreed that expiresOn will only be used for internal purposes (for Cron Jobs). To make this consistent with what Nnamdi told us and the Figma file, in cases where the user has not paid, then the due date should be used.

We might want to update the SMS message to reflect this too for past dates.

```suggestion
      }, is due on ${convertDateToShortHumanFormat(reminder.dueDate)}.`;
```